Wikipedia/Code/SavedViewController.swift

```diff
@@ -174,7 +173,8 @@ class SavedViewController: ViewController {
         edgesForExtendedLayout = .all
         
         pullToRefresh.addTarget(self, action: #selector(pulledToRefresh), for: .valueChanged)
-        
+        scrollView?.refreshControl = pullToRefresh
+
```

Comment:
@montehurd if we set `refreshControl` of this `scrollView`, it'll only work in the All articles view because when the view loads, the `scrollView` of the `SavedViewController` is set to that of `SavedArticlesViewController`'s (see line 85). I missed that we can use the refresh control inherited from `ColumnarCollectionViewController`, we were already doing that in `SavedArticlesViewController` and `ReadingListsViewController` (see the `isRefreshControlEnabled` flag)